mozjs/js/rust/build.rs

```diff
@@ -122,6 +122,8 @@ const WHITELIST_TYPES: &'static [&'static str] = &[
     "js::Class",
     "JS::CompartmentOptions",
     "JS::ContextOptions",
+    "js::DOMCallbacks",
```

Comment:
Want to squash this with b778ac1d0cddc25c79ae71b0008a1eeb55731844 and 42e2f2be47c9f9eac26a04a70ddd22c4dbe19b9a?